tests/unit/translator/resources/for_with_range.pml

```diff
@@ -24,6 +24,30 @@ inline Taskwithloop_0_transition(id)
         od;
       }
       ;
+      {
+        int x;
+        x = 1;
+        do
+        ::(x < 3)->
+          MyInteger_assign_value(global_state.taskwithloop.y, (global_state.taskwithloop.y + x));
+          x = (x + 1);
+        ::else->
+          break;
+        od;
+      }
+      ;
```

Comment:
Can we think about eliminating this one?

Reply:
This looks like separate task